ethcore/src/header.rs

```diff
@@ -16,13 +16,13 @@
 
 //! Block header.
 
+use std::cmp;
+use std::cell::RefCell;
 use util::*;
 use basic_types::{LogBloom, ZERO_LOGBLOOM};
 use time::get_time;
 use rlp::*;
```

Comment:
Probably not for this PR, but it'd be really nice if we could deglob everything apart from `util` (and ideally, even `util`). Better RLS refactoring support would make that way easier.